README.md

```diff
@@ -2,7 +2,7 @@
 
 [![CircleCI](https://circleci.com/gh/giantswarm/loki-app.svg?style=shield)](https://circleci.com/gh/giantswarm/loki-app)
 
-This chart provides a distributed loki setup based on this
+Giant Swarm offers Loki as a managed app. This chart provides a distributed loki setup based on this
```

Comment:
How to create relative link to ## About Giant Swarm managed apps section? 🙈

Reply:
set `#about-giant-swarm-managed-apps` as target

Reply:
thanks!